riak/tests/config/riak.conf

```diff
@@ -497,3 +497,11 @@ search.solr.jmx_port = 8985
 ## Acceptable values:
 ##   - text
 search.solr.jvm_options = -d64 -Xms1g -Xmx1g -XX:+UseStringCache -XX:+UseCompressedOops
+nodename = riak@172.18.0.2
```

Comment:
Why the update to the conf file ?

Reply:
oh, I didn't intend to commit it but also I didnt update it, it auto updates when running the test environment